probe-rs/targets/nRF52 Series.yaml

```diff
@@ -17,6 +21,7 @@ variants:
           page_size: 4096
           erased_byte_value: 255
   - name: nRF52832_xxAB
+    part: 0x000006
```

Comment:
Do the xxAB and xxAA variants have the same part number?

Reply:
I don't know tbh. I made an issue. We should track it in that.